JULIERAJ/KIDS-FIRST-Front
# Feat:Kid's info default page

Develop a default page for the "Kid's Info" section.

## Requirements
Develop a React component for the default page.
Default page appears when the user clicks on "Kid's Info" for the first time.
Ensure that the design matches the one on Figma.

## Test Steps
- Click on 'Kid's Info' in the sidebar and check if the landing page appears.

-![image](https://github.com/user-attachments/assets/6a6a8526-59c1-4386-a71f-b645c8cea669)

## Checklist
